include/storage/io.hpp

```diff
@@ -75,9 +82,15 @@ class FileReader
                                       ": Unexpected end of file");
             }
             throw util::exception("Error reading from " + filename + ": " + std::strerror(errno));
+            
         }
     }
 
+    template <typename T> void ReadInto(std::vector<T> &target)
+    {
+        ReadInto(target.data(), target.size());
+    }
```

Comment:
Should there be a similar function for strings? See comment above.

(Or even more generic)